doc/radosgw/multisite.rst

```diff
@@ -71,15 +71,16 @@ Functional Changes from Infernalis
 ==================================
 
 Beginning with Kraken, each Ceph Object Gateway can be configured to work in an
-active-active zone configuration, allowing for writes to non-master zones.
+active-active zone configuration. This makes it possible to write to non-master
```

Comment:
At both of these loci I think 'mode' fits better than 'configuration', ymmv.
Non-blocking.

Reply:
I'll chew on it for a day while I make other small alterations like this one. My very first thought is that "mode" is a word pretty much claimed by vi and vim, but my very first thought has been wrong before and you do know the pragmatics better than I.

Reply:
"Configuration" to me is something static:  `ceph.conf`, the mon's central config DB, even runtime injections.  "Mode" connotes the fashion in which things happen dynamically, like the two weeks I once spent in night mode, or the occasions when I'm in single-parent mode.

That said, use by `vi` alone is reason to chose a different word ;)

>"Droids, I want to lose.  I mean, I want
to lose.  I want to see line editors on CRTs and nulls in my files.

Reply:
1. I think I'm convinced.
2. That is an obscure Alice's Restaurant-adjacent reference. You are the James Joyce of the Ceph documentation.
3. Are we on different text-editor-based sides of a religious war?